VisualPinball.Engine/VPT/Table/Table.cs

```diff
@@ -251,6 +254,10 @@ public class Table : Item<TableData>, IRenderable
 
 		#endregion
 
+		public void Init(Table table)

```

Comment:
Is it needed ?

Reply:
It's part of `IPlayable`, so yes. Otherwise there would be a lot of refactoring for not much benefit.